docs/csharp/TestData.cs

```diff
@@ -1,9 +1,5 @@
 ﻿using FsCheck;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
```

Comment:
There are a bunch of cleanup related changes, mostly in docs. Worth just submitting these as a separate PR to master?